xbmc/music/Album.h

```diff
@@ -71,6 +71,8 @@ class CAlbum
   }
 
   std::string GetArtistString() const;
+  const std::vector<std::string>& GetAlbumArtist() const;
+  const std::vector<std::string>& GetMusicBrainzAlbumArtistID() const;
```

Comment:
Thanks for the correction @Jalle19 I will update.
@evilhamster not sure which way to go with the naming, as you point out none of it is pretty, hence the compromise